project/raft/project.json

```diff
@@ -0,0 +1,23 @@
+{
+    "maintainer": [
```

Comment:
All maintainers and committers of raft project should also be in tikv project.

Reply:
Make sense. Updating...

Reply:
Done.

Reply:
https://github.com/tikv/community/pull/122/commits/6fff113a394babcdb3f6a753b1eb9ee386146c53